lib/handlers/formatting.js

```diff
@@ -111,4 +111,23 @@ export class FormattingHandler {
             });
         }
     }
+
+    async internal_format(formatterName, style, source) {
```

Comment:
typically our style is to use `pascalCase` for functions - would you mind renaming this to `internalFormat` ? Thank you!